### PR TITLE
Update the module md file

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

### CODE REVIEW

File: cache-ballerina/src/cache/Module.md [hunk 1]
```diff
@@ -17,16 +17,16 @@ public type AbstractCache abstract object {
 };
 ```
 
-The `cache:AbstractEvictionPolicy` object has the common APIs for the cache eviction functionalities. Custom implementations of the eviction policy can be done by maintaining the `cache:LinkedList` data structure according to the eviction algorithm.
+The `cache:AbstractEvictionPolicy` object has the common APIs for the cache eviction functionalities. Custom implementations of the eviction policy can be done by maintaining the `cache:LinkedList` object according to the eviction algorithm.
```
Comment (ldclakmal) [inline cache-ballerina/src/cache/Module.md:20]:
```suggestion
The `cache:AbstractEvictionPolicy` class has the common APIs for the cache eviction functionalities. Custom implementations of the eviction policy can be done by maintaining the `cache:LinkedList` class according to the eviction algorithm.
```

Comment (kalaiyarasiganeshalingam) [reply cache-ballerina/src/cache/Module.md:20]: `cache:AbstractEvictionPolicy` is changed from abstract object to object. @ldclakmal IMO Better to continue to call those as `object`.

File: cache-ballerina/src/cache/Module.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ There are 2 mandatory scenarios and 1 optional scenario in which a cache entry g
 
 The main benefit of using the `cleanupIntervalInSeconds` (optional) property is that the developer can optimize the memory usage while adding some additional CPU costs and vice versa. The default behaviour is the CPU-optimized method.
 
-The concept of the default `cache:Cache` object is based on the Ballerina `map` data structure and the `cache:LinkedList` data structure. The key of the map entry would be a string and the value of the map entry would be a node of the linked list.
+The concept of the default `cache:Cache` object is based on the Ballerina `map` data structure and the `cache:LinkedList` object. The key of the map entry would be a string and the value of the map entry would be a node of the linked list.
```
Comment (ldclakmal) [inline cache-ballerina/src/cache/Module.md:64]:
```suggestion
The concept of the default `cache:Cache` class is based on the Ballerina `map` data structure and the `cache:LinkedList` class. The key of the map entry would be a string and the value of the map entry would be a node of the linked list.
```

File: cache-ballerina/src/cache/Module.md (hunk 1 above)
Comment (ldclakmal) [reply cache-ballerina/src/cache/Module.md:20]: Yes. Sorry, I missed that. Updated the comment.

**Review by ldclakmal — APPROVED** [review]

**Pull request merged**